src/helm.bash

```diff
@@ -24,18 +25,19 @@ helm_template() {
   output_file="${tmp_write_dir}/templates.yaml"
 
   # NOTE: eval is used due to helm not liking empty template_opts
-  eval "helm template ${template_opts} ${chart_dir} &> ${output_file}"
+  return_message=$(eval "helm template ${template_opts} ${chart_dir} &> ${output_file}")
 
-  # Safety check to make sure nothing above silently failed
-  if [[ ! -s "${output_file}" ]] || [[ $(wc -c "${output_file}" | awk '{print $1}') -le 1 ]] ; then
-    fail "# FATAL-ERROR: (helm.bash): File is empty: '${output_file}'" || return $?
+  # Safety check 
+  if [[ $? -eq 1 ]] ; then
```

Comment:
I'm thinking you probably want to change this to `[[ $? -ne 0 ]]` or `[[ $? -gt 0 ]]` in order to catch all possible failures from `helm template` (unless you know that `1` is the only error code).